libraries/AP_NavEKF/AP_Nav_Common.h

```diff
@@ -52,7 +52,7 @@ union nav_gps_status {
         bool bad_horiz_drift    : 1; // 5 - true if the GPS horizontal position is drifting (this check assumes vehicle is static)
         bool bad_hdop           : 1; // 6 - true if the reported HDoP is insufficient to start using GPS
         bool bad_vert_vel       : 1; // 7 - true if the GPS vertical speed is too large to start using GPS (this check assumes vehicle is static)
-        bool bad_fix            : 1; // 8 - true if the GPS is not providing a 3D fix
+        bool bad_fix            : 1; // 8 - true if the GPS is not providing the minimum required fix
```

Comment:
Bad fix is still 3D (same comment for the other file in which this comment was changed)

Reply:
Done